plugins/modules/mso_schema_site_external_epg.py

```diff
@@ -16,46 +16,64 @@
 short_description: Manage External EPG in schema of sites
 description:
 - Manage External EPG in schema of sites on Cisco ACI Multi-Site.
-- This module can only be used on versions of MSO that are 3.3 or greater.
+- This module can only be used on versions of MSO that are 3.3 or greater
 author:
 - Anvitha Jain (@anvitha-jain)
 options:
   schema:
     description:
-    - The name of the schema.
+    - The name of the schema
     type: str
     required: true
   template:
     description:
-    - The name of the template to change.
+    - The name of the template to change
     type: str
     required: true
   l3out:
```

Comment:
Should we introduce as separate input strings instead of dictionary in order to avoid breaking change? So l3out ( with alias l3out_name), l3out_schema, l3out_template as string inputs.

Reply:
Added back as discussed